### PR TITLE
feat: implement generic feed content translations

### LINKED ISSUE #1166: Feat: user friendly channel names
## Define your request
We'd better to have users friendly channel names to show within the left drawer. 

With the new filter categorization, user filters a category and then clicks on a marker. When the left drawers opens,  user may have a confusing channel name. To not confuse the user, we need user friendly channel names. 

For example, When user filters the `enkaz`, the channel name may have `Teyit_enkaz`.  It would be great to have Teyitli Enkaz Verisi or Enkaz (Teyitli) for the Turkish language options. 

> -- fill in the line below:
** discord username: @sercanerhan#0543 **

image

### LINKED ISSUE #1178: feat: Renaming & Consolidating Channel Names
## Define your request
> Define the feature you're requesting clearly and briefly.

Within this exercise, we will show user-friendly names for our data channels. Additionally, we will also change some of the icons as well to consolidate them.

Please check the `Channel_Name_TR` and `Channel_Name_EN` columns for the expected user-friendly channel names. 

**Changed (consolidated) icons:**
### Pharmacy
<!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}-->

turk_eczane | images/icon-15.png
-- | --
eczane_excel | images/icon-15.png

### Food
<!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}-->

sahra_mutfak | images/icon-11.png
-- | --
sicak_yemek | images/icon-11.png
adana_yemek | images/icon-11.png
malatya_yemek | images/icon-11.png



Please find all the data in below table:

| Channel Name                | Icon               | Reason                                                          | Source                                                                                                                                                                                                                                                                                                                                                                                                         | Status      | Last Update      | Type        | Channel_Name_TR                                         | Channel_Name_EN                                     |
| --------------------------- | ------------------ | --------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- | ---------------- | ----------- | ------------------------------------------------------- | --------------------------------------------------- |
| ahbap_location              | 8, 2, 9 ,10        | guvenli-noktalar, hayvanlar-icin-tedavi, saglik,konaklama,erzak | [https://www.google.com/maps/d/u/0/viewer?mid=1aQ0TJi4q_46XAZiSLggkbTjPzLGkTzQ&hl=tr&fbclid=PAAabyVDiTmFsFSPkcoAfAd9pcy5poekIm5LfUOndiuxmCXEMq4By3ObMr-_8&ll=39.43583732609636%2C38.94185249189591&z=6](https://www.google.com/maps/d/u/0/viewer?mid=1aQ0TJi4q_46XAZiSLggkbTjPzLGkTzQ&hl=tr&fbclid=PAAabyVDiTmFsFSPkcoAfAd9pcy5poekIm5LfUOndiuxmCXEMq4By3ObMr-_8&ll=39.43583732609636%2C38.94185249189591&z=6) | Production  | 20.02.2022 23:13 | GMaps       | Ahbap Lokasyonları                                      | Ahbap Locations                                     |
| babala                      | \-                 | \-                                                              | stk                                                                                                                                                                                                                                                                                                                                                                                                            | Production  | one timer        | Spreadsheet | Babala Verileri                                         | Babala Data                                         |
| depremio                    | \-                 | \-                                                              | outsource                                                                                                                                                                                                                                                                                                                                                                                                      | Development | every post       |             | [Deprem.io](http://deprem.io/) Çağrıları                | deprem.io Calls                                     |
| depremihtiyaccom            | images/icon-17.png | enkaz,erzak,genel                                               | [https://depremihtiyac.com](https://depremihtiyac.com/)                                                                                                                                                                                                                                                                                                                                                        | Development | 15.02.2023 19:11 | Spreadsheet | [depremihtiyac.com](http://depremihtiyac.com/) Verileri | [depremihtiyac.com](http://depremihtiyac.com/) Data |
| discord                     | \-                 | \-                                                              | social media                                                                                                                                                                                                                                                                                                                                                                                                   | Production  | every post       | Live        | Discord Çağrıları                                       | Discord Calls                                       |
| diyaliz_merkezleri          | images/icon-24.png | saglik,yardim                                                   | [https://www.google.com/maps/d/u/0/viewer?mid=1lmvt0ilt1gDnpy7tWTq8sKTwGIgBkZU&ll=38.73197551213188%2C35.60309690000001&z=6](https://www.google.com/maps/d/u/0/viewer?mid=1lmvt0ilt1gDnpy7tWTq8sKTwGIgBkZU&ll=38.73197551213188%2C35.60309690000001&z=6)                                                                                                                                                       | Development | 20.02.2022 23:13 | GMaps       | Diyaliz Merkezleri                                      | Dialysis Centers                                    |
| turk_eczane                 | images/icon-15.png | saglik                                                          | [https://www.google.com/maps/d/u/0/viewer?mid=1Ahwvt79jUCCRpn4cJlukvVUCT_8zeew&g_ep=CAISBjYuNTEuNhgAIN1iQgJUUg%3D%3D&g_st=ic&ll=38.33469995178001%2C37.28529207933022&z=7](https://www.google.com/maps/d/u/0/viewer?mid=1Ahwvt79jUCCRpn4cJlukvVUCT_8zeew&g_ep=CAISBjYuNTEuNhgAIN1iQgJUUg%3D%3D&g_st=ic&ll=38.33469995178001%2C37.28529207933022&z=7)                                                           | Production  | 20.02.2022 23:13 | GMaps       | Eczane                                                  | Pharmacy                                            |
| eczane_excel                | images/icon-15.png | saglik                                                          | [https://docs.google.com/spreadsheets/u/1/d/1Rjno8SF1tqFscHtk4Y6RCFt49WjP25Uo835rj6MTji4/htmlview](https://docs.google.com/spreadsheets/u/1/d/1Rjno8SF1tqFscHtk4Y6RCFt49WjP25Uo835rj6MTji4/htmlview)                                                                                                                                                                                                           | Development | 15.02.2023 19:11 | Spreadsheet | Eczane                                                  | Pharmacy                                            |
| gecici_barinma_gida_dagitim | images/icon-12.png | yemek,su,barinma                                                | [https://www.google.com/maps/d/u/0/viewer?mid=1PF6vNpvhfxhSaVvjD829J_ofJsYxNsM&fbclid=PAAaYs1qO1i3SN0UH0EMblOYrKk78d4qr7Ox8UREZsvpsonVxbrTXmkyzTWLc&ll=37.3437253445264%2C38.134183699999994&z=6](https://www.google.com/maps/d/u/0/viewer?mid=1PF6vNpvhfxhSaVvjD829J_ofJsYxNsM&fbclid=PAAaYs1qO1i3SN0UH0EMblOYrKk78d4qr7Ox8UREZsvpsonVxbrTXmkyzTWLc&ll=37.3437253445264%2C38.134183699999994&z=6)             | Development | 20.02.2022 23:13 | GMaps       | Geçici Barınma ve Gıda Dağıtım                          | Temporary Housing and Food Distribution             |
| guvenli_yerler_oteller      | images/icon-16.png | barinma                                                         | [https://www.google.com/maps/d/u/0/viewer?mid=1Rb3vfg521UFHrpDjJyP5m_awZzdmf08&ll=38.329008127119195%2C39.122894329271176&z=6](https://www.google.com/maps/d/u/0/viewer?mid=1Rb3vfg521UFHrpDjJyP5m_awZzdmf08&ll=38.329008127119195%2C39.122894329271176&z=6)                                                                                                                                                   | Production  | 20.02.2022 23:13 | GMaps       | Güvenli Yerler ve Oteller                               | Safe Places and Hotels                              |
| hastahane_locations         | \-                 | saglik                                                          | [https://docs.google.com/spreadsheets/d/1a785CPjKg0cVND2rl2dGnwb7q_1mPJhwWJ8UODCcttw/edit#gid=0](https://docs.google.com/spreadsheets/d/1a785CPjKg0cVND2rl2dGnwb7q_1mPJhwWJ8UODCcttw/edit#gid=0)                                                                                                                                                                                                               | Production  | 15.02.2023 19:11 | Spreadsheet | Hastane Lokasyonları                                    | Hospital Locations                                  |
| psikolojik_destek           | images/icon-26.png | saglik                                                          | [https://www.google.com/maps/d/u/0/viewer?mid=1kOYF87Nivn3BElcrzWQWddYUfwdEFME&ll=41.05253407533925%2C28.98919186862801&z=16](https://www.google.com/maps/d/u/0/viewer?mid=1kOYF87Nivn3BElcrzWQWddYUfwdEFME&ll=41.05253407533925%2C28.98919186862801&z=16)                                                                                                                                                     | Development | 20.02.2022 23:13 | GMaps       | Psikolojik Destek Merkezleri                            | Psychological Support Centers                       |
| tahliye_noktalari           | images/icon-23.png | lojistik,genel,yardim                                           | https://www.google.com/maps/d/u/0/viewer?mid=1xyEQW-Jyy0Qi-4kcj-VwnpKatLP0eHU&ll=37.30098042230212%2C37.73058195&z=8                                                                                                                                                                                                                                                                                           | Development | 20.02.2022 23:13 | GMaps       | Tahliye Noktaları                                       | Evacuation Locations                                |
| teyit_enkaz                 | images/icon-27.png | enkaz,genel,yardim                                              | [https://www.google.com/maps/d/u/0/viewer?mid=1vlyRlqckMKbN8eVuMqaWz__VTiQ1oNI&ll=36.34304859995416%2C36.156976650000004&z=9](https://www.google.com/maps/d/u/0/viewer?mid=1vlyRlqckMKbN8eVuMqaWz__VTiQ1oNI&ll=36.34304859995416%2C36.156976650000004&z=9)                                                                                                                                                     | Development | every 15 min     | GMaps       | Teyitli Enkaz Verileri                                  | Verified Rescue Data                                |
| teyit_yardim                | images/icon-28.png | enkaz,genel,yardim,barinma,yemek,su                             | [https://www.google.com/maps/d/u/0/viewer?mid=1M0ReZpNMK4dIRywyX5EMZES4zxrORkE&ll=36.7548629%2C36.23743940000001&z=9](https://www.google.com/maps/d/u/0/viewer?mid=1M0ReZpNMK4dIRywyX5EMZES4zxrORkE&ll=36.7548629%2C36.23743940000001&z=9)                                                                                                                                                                     | Development | every 15 min     | GMaps       | Teyitli Yardım Verileri                                 | Verified Relief Aid Data                            |
| twitter                     | \-                 | \-                                                              | social media                                                                                                                                                                                                                                                                                                                                                                                                   | Production  | every 30sec      | Live        | Twitter Yardım Çağrıları                                | Twitter Relief Aid Calls                            |
| uda_yardim                  | images/icon-20.png | genel,yardim,yemek,su,barinma,enkaz                             | [https://www.google.com/maps/d/viewer?mid=1u-cwFL0wa723ME4LN7ZMoIHwiEHmdCs](https://www.google.com/maps/d/viewer?mid=1u-cwFL0wa723ME4LN7ZMoIHwiEHmdCs)                                                                                                                                                                                                                                                         | Development | 20.02.2022 23:13 | GMaps       | Üniversite Dayanışma Ağı                                | University Solidarity Network                       |
| uydu                        | \-                 | \-                                                              | discord developers                                                                                                                                                                                                                                                                                                                                                                                             | Production  | one timer        | Live        | Uydu Görüntüsü Analizi                                  | Satellite Image Analysis                            |
| veteriner                   | images/icon-19.png | saglik                                                          | [https://www.google.com/maps/d/u/0/viewer?mid=1quGS8qonhKrGMr0aqfrNvLi9GzvYE0I&ll=0%2C0&z=6](https://www.google.com/maps/d/u/0/viewer?mid=1quGS8qonhKrGMr0aqfrNvLi9GzvYE0I&ll=0%2C0&z=6)                                                                                                                                                                                                                       | Development | 20.02.2022 23:13 | GMaps       | Veteriner Lokasyonları                                  | Veterinary locations                                |
| sahra_mutfak                | images/icon-11.png | yemek, su                                                       | [https://www.google.com/maps/d/u/0/viewer?mid=1LspPt4CpxG6krYf2ABBZISBbqd4SZs0&ll=37.226353761572526%2C37.777161299999996&z=8](https://www.google.com/maps/d/u/0/viewer?mid=1LspPt4CpxG6krYf2ABBZISBbqd4SZs0&ll=37.226353761572526%2C37.777161299999996&z=8)                                                                                                                                                   | Production  | 20.02.2022 23:13 | GMaps       | Yemek Dağıtım Lokasyonları                              | Food Distribution Locations                         |
| sicak_yemek                 | images/icon-11.png | yemek,su                                                        | [https://www.google.com/maps/d/u/0/viewer?mid=1Fthqb2VG-zK_Fu06lo1yPFAeRCyPfNs&ll=37.28762315063459%2C37.95504420000003&z=8](https://www.google.com/maps/d/u/0/viewer?mid=1Fthqb2VG-zK_Fu06lo1yPFAeRCyPfNs&ll=37.28762315063459%2C37.95504420000003&z=8)                                                                                                                                                       | Production  | 20.02.2022 23:13 | GMaps       | Yemek Dağıtım Lokasyonları                              | Food Distribution Locations                         |
| adana_yemek                 | images/icon-11.png | yemek,su                                                        | [https://www.google.com/maps/d/u/0/viewer?mid=1jEw7Qe2Z7SXH8a4rMx9bso1hycYi3vc&ll=36.994885473974875%2C35.40509821285594&z=11](https://www.google.com/maps/d/u/0/viewer?mid=1jEw7Qe2Z7SXH8a4rMx9bso1hycYi3vc&ll=36.994885473974875%2C35.40509821285594&z=11)                                                                                                                                                   | Development | 20.02.2022 23:13 | GMaps       | Yemek Dağıtım Lokasyonları                              | Food Distribution Locations                         |
| malatya_yemek               | images/icon-11.png | yemek,su                                                        | [https://www.google.com/maps/d/u/0/viewer?mid=1ah8VtFf5cGFsiS_KaHkadEzWehbLmWc&ll=38.463392440965414%2C38.311256550000024&z=11](https://www.google.com/maps/d/u/0/viewer?mid=1ah8VtFf5cGFsiS_KaHkadEzWehbLmWc&ll=38.463392440965414%2C38.311256550000024&z=11)                                                                                                                                                 | Development | 20.02.2022 23:13 | GMaps       | Yemek Dağıtım Lokasyonları                              | Food Distribution Locations                         |

-- fill in the line below:
** discord username: @sercanerhan#0543 **

## Define the alternatives you have in mind
Provide a clear and brief explanation to any alternative solution or feature you have in mind.

## Extra context
Add some extra context or screenshot pertaining to said feature request.

### PR DESCRIPTION
**Pull request by oguzergul**
# Description

This pull request adds i18n support for generic channel content titles. Additionally, I have included the channels malatya_yemek and adana_yemek, both of which are used for food-related content. The sahra_locations channel is also being used for food-related content."

**discord username: afetharita#0001**

**closes #1178 **
**closes #1166 **

Please describe your changes. Also describe your aim and content. Do not forget to list the dependencies required caused by those changes.

## Things to check before creating a PR
- [ ] I have inspected my topic, checked.
- [ ] If it is a core feature, executed detailed tests.

## Creating PR rules
- [ ] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [ ] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [ ] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [ ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [ ] Clean commit history: The commits where the changes are made should be clear and organized.
- [ ] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [ ] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

# How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by sercanerhan** [comment]
Couldn't validate the `Ahpab` with the proposed one. It should be `Ahbap Locations`.

image

**Comment by oguzergul** [comment]
@sercanerhan this is draft pr :)

**Comment by sercanerhan** [comment]
may bad 🤦‍♂️ @oguzergul

**Comment by eraygundogmus** [comment]
@oguzergul please fix build err

**Comment by oguzergul** [comment]
> @oguzergul please fix build err

fixed 👌🏼

File: public/locales/en/home.json [hunk 1]
```diff
@@ -95,6 +95,27 @@
         "closeBy": {
             "title": "Other records",
             "details": "You can click the below records to access the other records in the <b>same location</b>"
+        },
+        "channels": {
```
Comment (eraygundogmus) [inline public/locales/en/home.json:99]: prettierdan geçmemiş sanki

File: public/locales/tr/home.json [hunk 2]
```diff
@@ -97,6 +97,28 @@
         "closeBy": {
             "title": "Diğer kayıtlar",
             "details": "Aşağıdaki seçeneklere tıklayarak <b>aynı konumdaki</b> diğer kayıtlara ulaşabilirsiniz."
+        },
+        "channels": {
+            "ahbap":"Ahbap Lokasyonları",
```
Comment (eraygundogmus) [inline public/locales/tr/home.json:102]: burası da aynı şekilde prettierdan geçmemiş gibi

**Review by eraygundogmus — CHANGES_REQUESTED** [review]

**Comment by eraygundogmus** [comment]
++ you got conflicts

File: public/locales/en/home.json (hunk 1 above)
Comment (usirin) [reply public/locales/en/home.json:99]: bana da oyle geldi.

Comment (oguzergul) [reply public/locales/en/home.json:99]: .prettierignore dosyasında public klasörü ignore edilmiş. madem öyle bu dosyayı kaldırıyorum ignore dosyasyından.

**Comment by oguzergul** [comment]
> ++ you got conflicts

prettier issuesundan dolayı pr ı içeri almadığımız  ve üzerine hook refactor edildiği için boru gibi conflict olmuş. elimize sağlık 🙏🏼 (well thought pros-cons)

**Comment by eraygundogmus** [comment]
> > ++ you got conflicts
> 
> prettier issuesundan dolayı pr ı içeri almadığımız ve üzerine hook refactor edildiği için boru gibi conflict olmuş. elimize sağlık 🙏🏼 (well thought pros-cons)

kızdın mı :(

**Comment by eraygundogmus** [comment]
@oguzergul bro? what are these conflicts? are you even a real developer?

**Comment by sercanerhan** [comment]
Found `Yardım İçeriği` & `Ahbap` in channel names which are not respect to task description. Additionally, yemek icons are still different.

I've checked it since this task has the `waiting-test` label.  @oguzergul could you please guide if I need to use a different URL?

**Comment by sevilayerkan** [comment]
As test team we encountered couple of problems:

1) We are getting invalid icons like in the ss
image
image

2) We could not see any data coming from depremihtiyac.com channel

**Comment by oguzergul** [comment]
> As test team we encountered couple of problems:
> 
> 1. We are getting invalid icons like in the ss
>    image
>    image
> 2. We could not see any data coming from depremihtiyac.com channel

Please add the relevant test url to your comments so that we can try the same problems again. I also just threw another commit. He must have solved some of you. Can you check? @sevilayerkan

**Comment by sercanerhan** [comment]
tested the deployment - Feb 22, 2023 at 7:31PM (UTC) 👍  regarding to the issues **https://github.com/acikkaynak/deprem-yardim-frontend/issues/1178 ** and **https://github.com/acikkaynak/deprem-yardim-frontend/issues/1166 **

**Comment by sevilayerkan** [comment]
I added related url in comment. Also i checked again did not see first problem in the last vercel. @oguzergul

**Comment by oguzergul** [comment]
depremyardim data has not any relation with this pr. am I wrong? @sercanerhan @sevilayerkan

**Comment by sevilayerkan** [comment]
We checked out depremyardım datas because this pr covers the issue #1178 @oguzergul

**Comment by oguzergul** [comment]
@sevilayerkan  depremihtiyac.com not returning any data yet. so it is normal to not see any value.

**Pull request merged**